schema.sql

```diff
@@ -5,5 +5,5 @@ CREATE TABLE animals(
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL
-);
+    weight_kg DECIMAL,
+    species VARCHAR(40));
```

Comment:
- [ ] It is better to alter the table than to directly ad the column

See example: `ALTER TABLE animals ADD species varchar(255);`

Reply:
in the requirment they said that I must add it to the schema.sql file
![image](https://user-images.githubusercontent.com/74744787/154542638-ceda0892-624f-4eba-8d09-1be9c9ca877e.png)

Reply:
Yeah, but you should use the alter just as in the example I gave you.